pages/about/about_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: About the .gov program
+permalink: /about/
+layout: layouts/info-page
+sidenav: true
+excerpt: Learn more about the .gov program.
+tags: about
+eleventyNavigation:
+  key: about
+  order: 1
+  title: About the .gov program
+---
+  
+
+## Who we are
+
+We are a team of program managers, technologists, and analysts. We help government organizations in the U.S. gain public trust by making .gov a well-known, reliable, and secure space online.
+
+The .gov program is part of the [Cybersecurity and Infrastructure Security Agency](https://www.cisa.gov/). 
+
+Check out [job openings on our team](#).
+
+## What we do
+
+The .gov team manages the .gov top-level domain.
+
+### .Gov domain name registration
+
+We manage domain name registration for .gov. We are the .gov registrar. (A registrar manages domain name registration.) You can only request and register .gov domain names through us. We make it easy for eligible government organizations to register .gov domains. We verify the identity of everyone who requests a .gov domain and we validate that their organization meets the criteria for having a .gov domain.
+
+### Authoritative source for all .gov domain names and registrants
+
+We manage the authoritative record of all .gov domain names and their registrants. We are the .gov registry. (A registry is a database that includes domain names registered in a top-level domain.) We ensure that .gov domain names resolve in the global domain name system (DNS). DNS maps easy-to-remember names to hard-to-recall numbers. This allows us to use, for example, cisa.gov instead of something like 104.106.178.124.
+
+### Critical infrastructure for governments at all levels
+
+.Gov domains are intertwined with access to public services. We make .gov a trusted, secure space by:
+
+- Using multi-factor authentication for all accounts in our domain registration and management system
+- Preloading all new domains. This action requires browsers to use a secure HTTPS connection with your website. This ensures that the content you publish is exactly what your visitors get.
+- Administering our [domain requirements](domains_requirements.md) to protect the integrity of .gov
+- Publishing the [complete list of .gov domains](#)
+- Recommending [security best practices](#) for .gov domain holders
+- Continuously improving how we secure the .gov namespace
+
+Follow our [product strategy and development activities on GitHub](https://github.com/cisagov/getgov/tree/main/docs/product). 
+
+## What we offer
```

Comment:
Non-blocking content thought: Should this section be out on the home page instead of over here because it feels like it's promoting the use of .gov? Or if it stays here I wonder if we should elevate these bullets to subheadings?

Reply:
I do have this info on "Benefits of a .gov domain" page too. Are you think it would replace the existing benefits or that we'd add a new section about our offerings to the home page?

I'll elevate the bullets to subheadings.